packages/app/cypress/e2e/index.cy.ts

```diff
@@ -1,5 +1,19 @@
 import defaultMessages from '@packages/frontend-shared/src/locales/en-US.json'
 
+function getPathForPlatform (posixPath: string) {
```

Comment:
Can we import this here instead?

https://github.com/cypress-io/cypress/pull/19979/files#diff-a6c768053589e872a87d60df5a0ff7e7017fcb6ca62792b820ac0c50989f06aaR1